wp-includes/js/src/site-icon-maskable/maskable-icon-controls.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Retrieves the translation of text.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-i18n/
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * This module allows you to create and use standalone block editors. ;)
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * This package includes a library of generic WordPress components
+ * to be used for creating common UI elements shared between screens
+ * and features of the WordPress dashboard.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-components/
+ */
+import {
+	// __experimentalHStack as HStack,
+	Flex,
+	FlexBlock,
+	FlexItem,
+	PanelBody,
+	ToggleControl,
+} from '@wordpress/components';
+
+/**
+ * Core Data is a data module intended to
+ * simplify access to and manipulation
+ * of core WordPress entities.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-core-data/
+ */
+import { store as coreStore, useEntityProp } from '@wordpress/core-data';
+
+/**
+ * WordPress’ data module serves as a hub
+ * to manage application state
+ * for both plugins and WordPress itself.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data/
+ */
+import { useSelect } from '@wordpress/data';
+
+export default function MaskableControls() {
+	const [siteIconMaskable, setSiteIconMaskable] = useEntityProp(
+		'root',
+		'site',
+		'site_icon_maskable'
```

Comment:
This is unexpectedly showing up as the label in the pre-save dialog when it should:

> ![image](https://user-images.githubusercontent.com/134745/222027448-1edffcc5-2fe0-418b-8ef9-18b684dd58fd.png)

Reply:
This is unfortunately a known bug in gutenberg, I'm going to ref. the issue here, when I'll find it again ;)